airflow/www/views.py

```diff
@@ -2280,10 +2280,13 @@ class VariableView(wwwutils.DataProfilingMixin, AirflowModelView):
     verbose_name_plural = "Variables"
     list_template = 'airflow/variable_list.html'
 
-    def hidden_field_formatter(view, context, model, name):
```

Comment:
> THe reason it can be invalid is that it is a hidden field which needs to be decrypted.
I think is the other way around, the reason it can be invalid is because it's not hidden so it gets decrypted - when hidden the format is `********` and there's not attempt to decrypt, avoiding any invalid fail case.